commands/configure/index.js

```diff
@@ -108,6 +106,27 @@ program
       const downloaded = await experienceDownload(null, {}, loadedConfig);
       if (downloaded) {
         logResult('success', 'Downloaded all experience resources!', 'green');
+      } else {
+        if (!hasBootstrapped(appInfo)) {
+          const { shouldBootstrap } = await inquirer.prompt([{
+            type: 'confirm',
+            name: 'shouldBootstrap',
+            message: `Do you want to bootstrap your experience for application ${appInfo.name}?`
+          }]);
+          if (shouldBootstrap) {
+            await experienceBootstrap({}, loadedConfig, appInfo);
+          } else {
+            if (!appInfo.ftueTracking) { // eslint-disable-line max-depth
+              appInfo.ftueTracking = [];
+            }
+            appInfo.ftueTracking.push({
+              name: 'experience',
+              version: 3,
+              status: 'skipped'
+            });
+            await api.application.update({ appilcationId: appInfo.id }, { appInfo });
```

Comment:
typo

Reply:
also, this is 'patch' not 'update'. And I'd only update ftueTracking, don't patch with the rest of the info